src/stories/temp-converter.tsx

```diff
@@ -0,0 +1,105 @@
+import { useState, useEffect } from "react";
+
+export type TempConverterProps = {
+    items: string[];
```

Comment:
Defining the temp units as props does not really make sense. You probably won't use this component in different places with different units. Those are just static values inside of the component.  
And if you would, "items" would not be a great name to chose since it doesn't tell what's actually inside there. ✨ Naming is important ✨